src/client/components/vision2/camera/model.ts

```diff
@@ -0,0 +1,54 @@
+import { observable } from 'mobx'
+
+import { Matrix4 } from '../../../math/matrix4'
+import { Vector2 } from '../../../math/vector2'
+import { Vector3 } from '../../../math/vector3'
+
+export class CameraParams {
+  @observable.ref Hcw: Matrix4
+  @observable.ref lens: Lens
+
+  constructor({ Hcw, lens }: { Hcw: Matrix4, lens: Lens }) {
+    this.Hcw = Hcw
+    this.lens = lens
+  }
+}
+
+export class Lens {
+  @observable.ref projection: Projection
+  @observable.ref focalLength: number
+  @observable.ref centre?: Vector2
+  @observable.ref distortionCoeffecients?: Vector2
+
+  constructor({ projection, focalLength, centre, distortionCoeffecients }: {
+    projection: Projection,
+    focalLength: number,
+    centre?: Vector2,
+    distortionCoeffecients?: Vector2
```

Comment:
Should these be optional? Are they defaulted somewhere?

Reply:
Yes! They are defaulted in the material: https://github.com/NUbots/NUsight2/pull/312/files#diff-59b40caddcdba31574cf9e3beea3bbc0R80

Otherwise I have to default them in every story, which seems unnecessary since they seem optional "corrections".